pegasus/emails/hoc_signup_2018_receipt_en.md

```diff
@@ -21,6 +21,9 @@ Help recruit more people from your school and community by [sending our sample e
 ### 3. Start planning your event
 Choose an [Hour of Code activity](https://<%= hourofcode %>/learn) for your classroom and [review this how-to guide](https://<%= hourofcode %>/how-to) for more information on getting started.
 
+### 4. Stock up on swag
+Order materials to help get students excited about your event by heading to the Code.org [Amazon store](https://www.amazon.com/stores/page/8557B2A6-EBF2-4C9F-95C5-C3256FBA0220). [Order posters](https://www.amazon.com/promocode/A3QAYNZUZTSSNQ) (and get an extra 50% off), Hour of Code kits, stickers, and more! But hurry, supplies are limited.   
```

Comment:
Does this markdown work okay despite the new lines?

Reply:
I think it just looks like a new line in this github preview because of the set page width. But if you "View file" or look at the raw file, there should be no new lines.

Reply:
Ah, looks like github renders .md files a bit different to other formats where it doesn't wrap.  Cool.